sw/airborne/arch/chibios/mcu_periph/spi_arch.c

```diff
@@ -487,6 +487,31 @@ void spi4_arch_init(void)
 }
 #endif
 
+#if USE_SPI6
+
+#if defined(STM32H7XX)
+// Local variables (in DMA safe memory)
+static IN_BDMA_SECTION(struct spi_init spi6_init_s) = {
```

Comment:
same question. Is it for all H7 boards ? only for Tawaki ? what about the other SPI or I2C peripherals ?

Reply:
yes, SPI6 is on BDMA.

Reply:
cf I2C4 comment